daemon/Jobs/UpdaterJob.cs

```diff
@@ -169,7 +260,8 @@ public void Perform()
                 }
                 catch (Exception exception)
                 {
-                    var msg = "A error occured while attemting to resolve the download link for the update: \n" + exception;
+                    var msg = "A error occured while attempting to resolve the download link for the update: \n" +
+                              exception;
                     _logger.LogError(msg);
```

Comment:
Please don't log exceptions like these, there are specific methods for it.

The LogError signature for this one should be `_logger.LogError(exception, "An error occured while attempting to resolve the download link for the update");`